cmd/oras/internal/option/remote_test.go

```diff
@@ -43,9 +45,213 @@ var testTagList = struct {
 	Tags: []string{"tag"},
 }
 
+// localhostServerCert is a PEM-encoded TLS cert with SAN IPs
+// "127.0.0.1" and "[::1]", expiring at Jan 29 16:00:00 2084 GMT.
+// adapted from src/crypto/tls:
+// go run generate_cert.go  --rsa-bits 4096 --host 127.0.0.1,::1,oras.land --ca --start-date "Jan 1 00:00:00 1970" --duration=1000000h
```

Comment:
Where can I find `generate_cert.go`?

Reply:
Will it be better to replace it with an `openssl` command?

Reply:
Clarified in comment